### ISSUE TITLE
Accessibility: "Aria-labelledby" has incorrect (ID) reference

### ISSUE THREAD

**Issue by elrayle** [cul-it/exhibits-library-cornell-edu#410]
## Description

Accessibility analysis reports error: `"Aria-labelledby" has incorrect (ID) reference`
Impacted pages: too many to count

Reference: 
* [Using aria-labelledby to provide a name for user interface controls](https://www.w3.org/WAI/GL/wiki/Using_aria-labelledby_to_provide_a_name_for_user_interface_controls)

## To Reproduce

Example of one instance:
* https://exhibits.library.cornell.edu/summers-yield/feature/matt-kilbane
* view-source and search for `aria-labelledby`

There is only one instance on this page (and likely all pages).

Code:
```
<div id="ajax-modal" class="modal fade" tabindex="-1" role="dialog" aria-labelledby="modal menu" aria-hidden="true">
```

## Expected behavior

The value of `aria-labelledby` should be en element id that exists on the page.

**Comment by elrayle** [comment]
## Exploration

### Observations:
* The offending line is created in [app/views/shared/_modal.html](https://github.com/projectblacklight/spotlight/blob/v2.13.0/app/views/shared/_modal.html.erb#L1).
* It doesn't appear that any text ever exists in this dev, but there are assets that reference it and may generate this on the fly for some pages.
* This file does not exist starting in Spotlight v3.0.0.x
* Spotlight v3 is still a release candidate

### Options:
* **OPTION 1** - Override the partial, fix the invalid id, add an element to hold text describing the modal.  I'm not sure what the text would be.
* **OPTION 2** - Override the partial to remove this unused structure.  Except that it might not be unused on all pages.

**Comment by kevinkidwell** [comment]
### Solutions:
1. Remove the modal entirely from the site. This is what was done for this same issue on ESMIS and has seen no adverse effects.

### To Review:
1. [Apples to Cider](https://exhibits.library.cornell.edu/cider/feature/old-fruit-new-light) has this issue on a gallery. Need to further dig into why only this one is showing this issue to make a solution recommendation.
